src/lib/scripts/createBundler.js

```diff
@@ -92,5 +97,14 @@ export default function(options) {
     }
   });
 
+  //configure externals
+  externals.forEach(external => {
+    if (Array.isArray(external)) {
```

Comment:
The `external()` method takes a string or an array of strings so I don't think you need the `if` stmt